### PR TITLE
Javascript/javascript1/week3

### PR DESCRIPTION
**Pull request by seenachacko**

### CODE REVIEW

File: javascript/javascript1/week3/script.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ]; 
+  const nameToRemove = "Ahmad";
+  for(let arrayIndex of names ){
+    if(nameToRemove === arrayIndex){
+        names.splice(arrayIndex ,1)
```
Comment (cameronjdavis) [inline javascript/javascript1/week3/script.js]: This removed "Peter", you need to remove "Ahmad".

```
Remaining array after removel:  Ahmad,Yana,kristina,Rasmus,Samuel,katrine,Tala
```
The problem is this `names.splice(arrayIndex ,1)`. `arrayIndex` holds the value "Ahmad". Instead, you need to get the array index where "Ahmad" is. Try this.
```
const index = names.indexOf(nameToRemove);
names.splice(index, 1);
```

File: javascript/javascript1/week3/script.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    return( travelInformation.destinationDistance/travelInformation.speed);
+  }
+  const travelTime = notThisFunctionName(travelInformation);
+  const timeinHhMm=timeConvertiontoHhMm( travelTime);
+  //refrece:how to covert decimal time to HH:mm  https://stackoverflow.com/questions/35460303/how-to-convert-decimal-hour-value-to-hhmmss/35460620
+  function timeConvertiontoHhMm(decimaltime){
+  let hrs = parseInt(Number(decimaltime));
+  let min = Math.round((Number(decimaltime)-hrs) * 60);
+  return(hrs+'hh:'+min+'mm');
+}
```
Comment (cameronjdavis) [inline javascript/javascript1/week3/script.js]: Make sure you use consistent spaces. It makes code easier to read. Are you using Visual Studio Code? If you are, you can tell VS code to Format Document.

```suggestion
function timeConvertiontoHhMm(decimaltime) {
  let hrs = parseInt(Number(decimaltime));
  let min = Math.round((Number(decimaltime)-hrs) * 60);
  return hrs+'hh:'+min+'mm';
}
```

File: javascript/javascript1/week3/script.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+console.log(notes);
+
+function getNote(id) {
+
+    for (const checkId of notes) {
+        if (checkId.id===id) {
+            let details = checkId;
+            return details;
+        }  
+    }
```
Comment (cameronjdavis) [inline javascript/javascript1/week3/script.js]: `checkId` is a confusing name.

```suggestion
    for (const note of notes) {
        if (note.id===id) {
            let details = note;
            return details;
        }  
    }
```

**Review by cameronjdavis — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week3/script.js (hunk 3 above)
Comment (seenachacko) [reply javascript/javascript1/week3/script.js]: thank you for your good feedbacks
I have updated the problem based on the suggestions.

**Pull request merged**